AdobeStockImageAdminUi/Test/Mftf/Section/AdminEnhancedMediaGalleryImageActionsSection.xml

```diff
@@ -9,5 +9,6 @@
 	xsi:noNamespaceSchemaLocation="urn:magento:mftf:Page/etc/SectionObject.xsd">
     <section name="AdminEnhancedMediaGalleryImageActionsSection">
         <element name="license" type="button" selector="[data-ui-id='action-license']"/>
+        <element name="contextMenu" type="block" selector="//div[@class='media-gallery-image']//ul[@class='action-menu _active']//li//a[@class='action-menu-item']"/>
```

Comment:
I'd rename this element to the `contextMenuItem` and move it to MediaGalleryUi (as this is the module introducing this menu)